appengine/helloworld-new-plugins/gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+##############################################################################
```

Comment:
Auto-generated? In that case, maybe add to `.gitignore`. If not auto-generated, add a license header.